CMakeLists.txt

```diff
@@ -182,7 +182,7 @@ set (SRCS
 
 esma_add_library (${this}
   SRCS ${SRCS}
-  DEPENDENCIES GFDL_fms_r8
+  DEPENDENCIES FMS_r8
```

Comment:
does it have to be caps?

Reply:
I think @tclune said this is consistent with how we handle external repos/libraries in our model elsewhere. `FVdycoreCubed_GridComp.git` is added as `@FVdycoreCubed_GridComp` and then referred to in CMake as `FVdycoreCubed_GridComp` I guess was the thought. So `FMS.git`=>`@FMS`=>`FMS_r4` and `FMS_r8`

But if there are good reasons not to do this, let me know so I can do the 7-repo change consistently and test it.

Reply:
Sorry - there was a bit of confusion when @mathomp4 and I were discussing this morning.   I meant that we should use caps for the mount point for the directory.  I.e. if we bring in repository   FOO then the mount point should be @FOO.

For the library name, I have less of an opinion.  Those are often lower case even for all caps repos.  (E.g., libpfunit.a for pFUnit which has the oddest capitalization you are likely to run across).

Reply:
I will lower case it

Reply:
I still might be confused by what @tclune so I need him to tell me exactly in person, I think